pkg/api/norman/customization/project/project_actions.go

```diff
@@ -281,13 +284,18 @@ func (h *Handler) setPodSecurityPolicyTemplate(actionName string, action *types.
 		}
 
 		clusterName := idParts[0]
-		cluster, err := h.ClusterLister.Get("", clusterName)
+		managementCluster, err := h.ClusterLister.Get("", clusterName)
 		if err != nil {
-			return fmt.Errorf("error retrieving cluster [%s]: %v", clusterName, err)
+			return fmt.Errorf("error retrieving management cluster [%s]: %v", clusterName, err)
```

Comment:
Noticed that we use `%w` in another place here to wrap errors, but not here or a couple of lines below. Fine to leave it not addressed, but ideally would like to go with `%w` for any new/updated code for errors.

Reply:
I can fix that when I push the rebase